<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fast Logo</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>222.jpg</normaloff>222.jpg</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>501</width>
      <height>311</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Input</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_input">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>501</width>
        <height>291</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>16</number>
      </property>
      <property name="columnCount">
       <number>24</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>45</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>21</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <item row="0" column="0">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="0" column="2">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="0" column="3">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="0" column="4">
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item row="0" column="5">
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item row="0" column="6">
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item row="0" column="7">
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item row="1" column="2">
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item row="1" column="3">
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item row="1" column="4">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="1" column="5">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="1" column="6">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="1" column="7">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item row="2" column="1">
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item row="2" column="2">
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item row="2" column="3">
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item row="2" column="4">
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item row="2" column="5">
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item row="2" column="6">
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item row="2" column="7">
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item row="3" column="0">
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item row="3" column="1">
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item row="3" column="2">
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item row="3" column="3">
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item row="3" column="4">
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item row="3" column="5">
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item row="3" column="6">
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item row="3" column="7">
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item row="4" column="0">
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item row="4" column="1">
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item row="4" column="2">
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item row="4" column="3">
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item row="4" column="4">
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item row="4" column="5">
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item row="4" column="6">
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item row="4" column="7">
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item row="5" column="0">
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item row="5" column="1">
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item row="5" column="2">
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item row="5" column="3">
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item row="5" column="4">
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item row="5" column="5">
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item row="5" column="6">
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item row="5" column="7">
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item row="6" column="0">
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item row="6" column="1">
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item row="6" column="2">
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item row="6" column="3">
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item row="6" column="4">
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item row="6" column="5">
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item row="6" column="6">
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item row="6" column="7">
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item row="7" column="0">
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item row="7" column="1">
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item row="7" column="2">
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item row="7" column="3">
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item row="7" column="4">
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item row="7" column="5">
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item row="7" column="6">
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item row="7" column="7">
       <property name="text">
        <string>6</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Output</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_output">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>501</width>
        <height>291</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>16</number>
      </property>
      <property name="columnCount">
       <number>24</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>45</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_gen">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate!</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_Paras">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>241</width>
      <height>291</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>paras</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Value</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Parameters:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_flushimg">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Flushimg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_curve">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="use curve paras to adjust pwm duties!"/>
    </property>
    <property name="text">
     <string>CurveAdjust</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>867</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="toolTip">
     <string extracomment="loading setting file or othes"/>
    </property>
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen_setting_file"/>
    <addaction name="actionSave_setting_file"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuInput">
    <property name="title">
     <string>Input</string>
    </property>
    <addaction name="actionLoadInputFile"/>
    <addaction name="actionSaveToCSV"/>
    <addaction name="actionSaveToXLSX"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuInput"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_setting_file">
   <property name="text">
    <string>Open setting file</string>
   </property>
  </action>
  <action name="actionSave_setting_file">
   <property name="text">
    <string>Save setting file</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionInputFile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>InputFile</string>
   </property>
  </action>
  <action name="actionInputFile_rl">
   <property name="text">
    <string>InputFile</string>
   </property>
  </action>
  <action name="actionLoadInputFile">
   <property name="text">
    <string>LoadInputFile</string>
   </property>
  </action>
  <action name="actionSaveToCSV">
   <property name="text">
    <string>SaveToCSV</string>
   </property>
  </action>
  <action name="actionSaveToXLSX">
   <property name="text">
    <string>SaveToXLSX</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
